selfdrive/test/process_replay/process_replay.py

```diff
@@ -10,6 +10,8 @@
 import capnp
 from tqdm import tqdm
 
+os.environ["SIMULATION"] = "1"
```

Comment:
I don't think we want to set this on import time. Would probably cause all kind of unexpected stuff to happen.